var/spack/repos/builtin/packages/py-py-cpuinfo/package.py

```diff
@@ -12,6 +12,7 @@ class PyPyCpuinfo(PythonPackage):
     homepage = "https://github.com/workhorsy/py-cpuinfo"
     pypi = "py-cpuinfo/py-cpuinfo-0.2.3.tar.gz"
 
+    version('6.0.0', sha256='7ffb31dea845b9f359b99bd5f7eea72dc70f852e0e34547d261a630f2b8c9c61')
```

Comment:
Latest version seems to be v8.0.0. Can you update the PR?

Reply:
I can, but I cannot promise anything regarding usability ;)